beta/nncf/tensorflow/sparsity/magnitude/algorithm.py

```diff
@@ -172,18 +158,15 @@ def _collect_all_weights(self):
         all_weights = []
         for wrapped_layer in collect_wrapped_layers(self._model):
             for weight_attr, ops in wrapped_layer.weights_attr_ops.items():
-                for op in ops.values():
-                    if isinstance(op, BinaryMask):
+                for op_name in ops.keys():
```

Comment:
Can you please refactor this according to the style guide?

Reply:
Done